index.html

```diff
@@ -7,6 +7,7 @@
 <body>
 	<div class="container">
 		<h1>YouTubers' Birthdays</h1>
+		<p>YouTubers on the list: <span class="count">0</span></p>
```

Comment:
Can it be in the center of the page?

Reply:
Yes it can. I'll change it

Reply:
![image](https://user-images.githubusercontent.com/71945723/229271941-d5e7f4b7-7997-4f59-8b92-6db2a6a90823.png)

Check it out now